### PR TITLE
mq-broker: upgrade versions

### PR DESCRIPTION
**Pull request by johncblandii**
## what
* Upgraded to the latest `terraform-aws-mq-broker`
* Updated the `providers` to match the common pattern
* Updated the module versions

## why
* The component was dated

### CODE REVIEW

File: modules/mq-broker/main.tf [hunk 1]
```diff
@@ -13,7 +13,7 @@ locals {
 
 module "mq_broker" {
   source  = "cloudposse/mq-broker/aws"
-  version = "0.14.0"
+  version = "2.0.1"
```
[the remark this answers is not in the document]
Comment (Nuru) [reply modules/mq-broker/main.tf:16]:
```suggestion
  version = "3.0.0"
```

**Comment by johncblandii** [comment]
@aknysh all comments addressed.

**Review by milldr — APPROVED** [review]

File: modules/mq-broker/providers.tf [hunk 2]
```diff
@@ -1,8 +1,14 @@
 provider "aws" {
```
Comment (milldr) [inline modules/mq-broker/providers.tf:1]: reset this to match https://github.com/cloudposse/terraform-aws-components/blob/main/mixins/providers.depth-1.tf

**Review by milldr — CHANGES_REQUESTED** [review]
resolve merge conflicts, but otherwise LGTM!

File: modules/mq-broker/providers.tf (hunk 2 above)
Comment (goruha) [reply modules/mq-broker/providers.tf]: Done

**Review by milldr — APPROVED** [review]

**Pull request merged**